<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CanPaddle</class>
 <widget class="QMainWindow" name="CanPaddle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1372</width>
    <height>681</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>2560</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CanPaddle</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>400</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>16777215</height>
         </size>
        </property>
        <column>
         <property name="text">
          <string notr="true"/>
         </property>
        </column>
        <item>
         <property name="text">
          <string>Networks</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ECUs</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Network nodes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Messages</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Signals</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1372</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionshow_grid"/>
    <addaction name="actionshow_hex"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionshow_grid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>show grid</string>
   </property>
  </action>
  <action name="actionshow_hex">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>show hex</string>
   </property>
   <property name="toolTip">
    <string>show message id in hex</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>CanPaddle</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionshow_grid</sender>
   <signal>toggled(bool)</signal>
   <receiver>tableWidget</receiver>
   <slot>setShowGrid(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>888</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
